### PR TITLE
local setup and basic dash app

### PR DESCRIPTION
**Pull request by nataliarizzi13**
That's what I did for this subtask:
- Installed python3
- Created virtual env
- Added .gitignore file
- Installed dash and pandas
- Added the code from dash documentation (just to have it as a placeholder)
- Added Procfile

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from dash import Dash, html, dcc, callback, Output, Input
+import plotly.express as px
+import pandas as pd
+
+df = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/gapminder_unfiltered.csv')
+
+app = Dash(__name__)
+
```
Comment (victorsarker) [inline app.py:8]: I think in order for the Procfile to work we need to add the line:
`server = app.server`

**Review by victorsarker — COMMENTED** [review]
All looks good, could you also add a requirements.txt file with the dash and pandas modules?

**Comment by nataliarizzi13** [comment]
@victorsarker changes required made. If everything looks ok, we'll be good to merge into the dev branch.

**Review by victorsarker — APPROVED** [review]
Perfect, thank you. Will merge this into the dev branch

**Pull request merged**